### ISSUE TITLE
make guest account more "persistant"

### ISSUE THREAD

**Issue by delmarle** [alvyxaz/barebones-masterserver#12]
Hey there, I was thinking about the guest account, everytime i relaunch the game it make a new account when i click use guest, why not making it more persistant by using something like this for login: guest_ + device unique id (https://docs.unity3d.com/ScriptReference/SystemInfo-deviceUniqueIdentifier.html)

that way we can retreive the guest account automaticaly.

**Comment by alvyxaz** [comment]
Hey!

This is a great suggestion! I wanted to do something similar earlier, but forgot about.

I'm not sure if it's making into V1.03, but I'm now planning to do this soon ;)

**Comment by delmarle** [comment]
yep no need to squeeze it into 1.03 its not something critical but just a "nice have" feature

**Comment by alvyxaz** [comment]
Alright, thanks again for the request ;)
